js/ddmlib/property_formatter.js

```diff
@@ -98,6 +110,11 @@ const formatProperties = function(root /* ViewNode */) {
         for (let i = 0; i < node.properties.length; i++) {
             node.namedProperties[node.properties[i].fullname] = node.properties[i];
         }
+
+        node.desc = node.name;
+        node.disablePreview = false
+        node.isVisible = node.visibility == VIEW_VISIBLE
+        node.nodeDrawn = node.isVisible
```

Comment:
does this run only for time-lapse data for normal data as well?

The logic here does not match what viewnode.js is doing

Reply:
Yes, this is running for normal data as well. I just updated the cl to match what viewnode.js was previously doing. Now the functionality should be identical.